homework-submission/myapp/src/components/addMenu.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import DatePicker from "react-datepicker";
+import moment from "moment";
+import "react-datepicker/dist/react-datepicker.css";
+import { observer, inject } from "mobx-react";
+
+@inject("todoStore")
+@observer
+class AddMenu extends React.Component {
+  state = {
+    name: "",
+    deadline: ""
+  };
+
+  handleChangeForm = ({ target: { name, value } }) => {
+    this.setState({ [name]: value });
+  };
+
+  handleChangeDate = date => {
+    this.setState({ deadline: date });
+  };
+
+  handleSubmitAdd = event => {
+    event.preventDefault();
+    this.props.todoStore.handleAdd(this.state.name, this.state.deadline);
+    this.setState({
+      name: "",
+      deadline: moment()
```

Comment:
Here you are setting it to be a `moment` while you have it by default as a string in your state.

I got this warning while starting the program:
<img width="914" alt="screenshot 2018-11-29 at 02 40 31" src="https://user-images.githubusercontent.com/25272822/49193663-2f50fb80-f380-11e8-9ba8-9383abb941e2.png">

with these both if no need for moment.js It's for a better performance.
By checking the Docs of [react-datepicker](https://www.npmjs.com/package/react-datepicker)

you will find out they [started using native js Date](https://www.npmjs.com/package/react-datepicker#momentjs) and you have the newest version there 😃